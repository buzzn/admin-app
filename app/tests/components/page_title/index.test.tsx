import React from 'react';
import renderer from 'react-test-renderer';
import PageTitle from 'components/page_title';

describe('page title tests', () => {
  const breadcrumbs = [{ id: 0, title: 'Title' }];
  const title = 'Page title';

  test('regular', () => {
    const tree = renderer.create(<PageTitle {...{ title, breadcrumbs }} />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  test('thin', () => {
    const tree = renderer.create(<PageTitle {...{ title, breadcrumbs, thin: 'true' }} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
