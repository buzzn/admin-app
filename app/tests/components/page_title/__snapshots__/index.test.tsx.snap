// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`page title tests regular 1`] = `
<div
  className="sc-bwzfXH cQsxgW row"
  thin={undefined}
>
  <div
    className="col-12"
  >
    <div
      className="sc-bdVaJa RchYe"
    >
      <ul>
        <li>
          <span>
            Title
          </span>
        </li>
      </ul>
    </div>
    <span
      className="sc-htpNat dpSnWL"
    >
      Page title
    </span>
  </div>
</div>
`;

exports[`page title tests thin 1`] = `
<div
  className="sc-bwzfXH cZZlKf row"
  thin="true"
>
  <div
    className="col-12"
  >
    <div
      className="sc-bdVaJa RchYe"
    >
      <ul>
        <li>
          <span>
            Title
          </span>
        </li>
      </ul>
    </div>
    <span
      className="sc-htpNat dpSnWL"
    >
      Page title
    </span>
  </div>
</div>
`;
